### ISSUE TITLE
Support wildcards for scp transfers

### ISSUE THREAD

**Issue by jorwex** [blinksh/blink#1189]
When attempting to use an asterisk in an scp request, Blink Shell's (13.5.10) logging is potentially misleading:

```
blink> scp user@host:/path/to/files/file-* .
scp: user@host:/path/to/files/file-*: No match
blink>
```

This implies that a connection was made to a host and no matching files were found, when in reality, (I believe that) no connection was attempted. 

Judging by:

https://github.com/blinksh/blink/blob/6705dea63973c4a70b5e9678b9d744eea784dd19/Resources/commandDictionary.plist#L292-L295

curl might be used internally for scp, and I do not believe curl supports wildcards.

Logging this feature request in case there is a simple solution for this. Perhaps using a native scp client can be baked into Blink Shell?

If it is not possible without major changes, then request that a more specific error message could be displayed when an asterisk is used, such as "Wildcards are not supported."

**Comment by carloscabanero** [comment]
This is done and should be available on next TF.

**Comment by carloscabanero** [comment]
This is now available on TF 13.90! Please check it out.

**Comment by jorwex** [comment]
Thanks! What is the time between when a TF is available and the equivalent release on the App Store?

**Comment by carloscabanero** [comment]
These are big changes, so we are keeping it open longer. Send us a dm on discord, Twitter or to hello at blink.sh if you want access to TF earlier.

**Comment by jorwex** [comment]
Just tested the fix in 14.0.5. Works great. I think this can be closed.
